src/containerUtils.js

```diff
@@ -0,0 +1,54 @@
+import invariant from 'invariant';
+import React from 'react';
+import Relay from 'react-relay';
+
+import BatchedRelayContainerBase from './BatchedRelayContainerBase';
+
+export function createBatchedRelayContainer(
+    childElement,
+    params,
+    routeName,
+    queryBatcher
+) {
+  const {queries, fragments} = queryBatcher.flush();
+  const route = {name: routeName, queries, params};
+
+  class BatchedRelayContainer extends BatchedRelayContainerBase {
+    static childElement = childElement;
+    static fragments = fragments;
+  }
+
+  return (
+    <Relay.RootContainer
```

Comment:
Ahh, I just didn't get this in yet. Wanted to start getting some feedback. Soon!